src/oop/oop1.py

```diff
@@ -17,3 +17,36 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+
```

Comment:
all of these classes can just have `pass` as the body.  Main goal was to make sure they are inheriting from the parent class which is taken care of inside the parenthesis of the class declaration.